_includes/blog-sidebar.html

```diff
@@ -20,5 +20,5 @@ <h2>Subscribe</h2>
           <h2>Contribute</h2>
           <p>The source for this blog can be found on <a href="https://github.com/bazelbuild/bazel-blog">GitHub</a>.</p>
           <h2>Archive</h2>
-          <p>Find the list of all posts on the <a href="/archive.html">Archive</a> page.</p>
+          <p>Looking for a post? Find it on the <a href="/archive.html">Archive</a> page.</p>
```

Comment:
Nit: I'd say "looking for a specific post" - that's why they go to the list, not just any post.

Reply:
Done